reference-architecture/aws-ansible/playbooks/roles/prerequisites/tasks/main.yaml

```diff
@@ -11,3 +11,13 @@
       - bash-completion
       - docker
       - python-httplib2
+  when: ansible_distribution == "RedHat"
+
+- name: Install the required rpms
+  yum: name=NetworkManager state=latest
+
+- name: Start NetworkManager
+  service:
+    name: NetworkManager
+    state: started
+    enabled: true
```

Comment:
Pre-requisites is missing, as some of the packages comes from epel and default aws centos does not have it enabled by default. This package in particular python-httplib2

````
---
- name: Install epel-release
  yum: name={{ item }} state=latest
  with_items:
      - epel-release
  when: deployment_type == "origin"

- name: Install the required rpms
  yum: name={{ item }} state=latest
  with_items:
      - wget
      - git
      - net-tools
      - bind-utils
      - iptables-services
      - bridge-utils
      - bash-completion
      - docker
      - python-httplib2
````